mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,209 @@
+# Getting Started
+
+Welcome to the "Getting Started" guide. This chapter will lead you through the initial steps of logging into the HPC-system and submitting your very first job. We'll also walk you through the process step by step using a practical example.
+
+In addition to this chapter, you might find these  [slides](https://www.ugent.be/hpc/en/training/2023/introhpc202306) to be useful resources.
+
+Before proceeding, read [the introduction to HPC](introduction.md) to gain an understanding of the HPC infrastructure and its terminology.
+
+### Getting Access
+
+To get access, visit <https://docs.hpc.ugent.be/account>.
+
+If you have not used linux before, now would be a good time to follow our [Linux Tutorial]().
+  
+#### A typical workflow looks like this:
+
+1.  Connect to the login nodes 
+3.  Transfer your files to the cluster 
+3.  Optional: compile your code and test it 
+4.  Create a job script and submit your job
+5.  Wait for job to be executed
+6.  Study the results generated by your jobs, either on the cluster or
+    after downloading them locally.
+
+We will walk through an illustrative workload to get you started. In this example, our objective is to train a deep learning model for recognizing hand-written digits (MNIST dataset).
+See the [example](https://github.com/EESSI/eessi-demo/tree/main/TensorFlow).
+
+### Getting Connected
+
+There are two options to connect
+
+- Using a terminal(ssh) (for power users) (see [First Time connection to the HPC infrastructure](connecting.md#first-time-connection-to-the-hpc-infrastructure))
+- [Using the HPC-Ugent web portal](web_portal.md#using-the-hpc-ugent-web-portal)
+
+{%- if OS == windows %}
+
+Considering your operating system is **{{OS}}**, it's recommented to utilize the UGent web portal. This platform offers a convenient way to upload files and gain shell access.
+
+Ensure you can get to a shell before proceeding with the next steps.
+
+See [shell access](web_portal.md#shell-access) when using the ugent portal or
+[connection to the hpc infrastructur](connecting.md#first-time-connection-to-the-hpc-infrastructure) when using a terminal.
+
+!!! Info
+
+    When having problems see the [connection issues](troubleshooting.md#sec:connecting-issues) section on the [troubleshooting page](troubleshooting.md).
+
+{%- else %}
+
+Considering your operating system is **{{OS}}**, it's recommented to make use of a terminal with ssh to get the most flexibility. 
```

Comment:
```suggested
Considering your operating system is **{{OS}}**, it's recommended to make use of a terminal with SSH to get the most flexibility.
```

Reply:
I'd say this is true for Linux users, but maybe not for macOS - I'm sure there's many researchers who are using macOS who have no idea on how to use a terminal...